lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -581,7 +587,15 @@ LogicalResult Deserializer::processArrayType(ArrayRef<uint32_t> operands) {
            << defOp->getName();
   }
 
-  typeMap[operands[0]] = spirv::ArrayType::get(elementTy, count);
+  spirv::ArrayType::LayoutInfo layoutInfo = 0;
```

Comment:
You could just do `spirv::ArrayType::LayoutInfo layoutInfo = typeDecorations.lookup(operands[0])`

Reply:
Thanks for mention this!